### PR TITLE
:bug: Filtering business service on copy assessment form

### PR DESCRIPTION
**Pull request by djzager**
Previously, the business service filter wasn't working on the copy assessment form. This makes it so we can filter results when copying assessments. It also removes the "" filter option.

Resolves https://issues.redhat.com/browse/MTA-34

### CODE REVIEW

File: client/src/app/pages/applications/components/bulk-copy-assessment-review-form/bulk-copy-assessment-review-form.tsx [hunk 1]
```diff
@@ -103,11 +103,9 @@ export const BulkCopyAssessmentReviewForm: React.FC<
       type: FilterType.multiselect,
       selectOptions: dedupeFunction(
         applications
+          .filter((app) => !!app.businessService?.name)
           .map((app) => app.businessService?.name)
-          .map((name) => ({
-            key: `BulkCopyAssessmentReviewForm-${name}`,
```
Comment (djzager) [inline client/src/app/pages/applications/components/bulk-copy-assessment-review-form/bulk-copy-assessment-review-form.tsx:108]: I verified I was still able to copy an assessment, so I'm not certain what impact this was supposed to have.

**Comment by djzager** [comment]
image

**Comment by gildub** [comment]
@djzager, please reference MTA issue (34) in PR description for follow-up.

**Comment by gildub** [comment]
@djzager, thanks for your help.
The filtering is needed to avoid empty entries for applications with no business service assigned.

The issue for business services filters no working was because the application key's name returned was not the application name.

**Comment by ibolton336** [comment]
@djzager @gildub Thanks for your help on this. Will merge as soon as I have a working hub to test against.

**Review by ibolton336 — APPROVED** [review]
LGTM!

**Pull request merged**